gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/api/AbstractPluginHandler.java

```diff
@@ -61,7 +62,9 @@ public void handle(Plugin plugin) {
                     if (pluginDeploymentLifecycle.isDeployable(pluginDeploymentContextFactory.create())) {
                         handle(plugin, pluginClass);
                     } else {
-                        logger.warn("Plugin {} can't be deployed", plugin.id());
+                        ((PluginImpl) plugin).setDeployed(false);
+                        handle(plugin, pluginClass);
+                        logger.warn("Plugin {} detected but not deployed", plugin.id());
```

Comment:
I am not sure to understand this log as you are calling `handle(plugin, pluginClass);` as if it was deployed. Could you explain?

Reply:
Yeah maybe the log is not perfect. What I was trying to tell here is that your plugin is now "visible" using the word detected but with a particular state "not deployed". 
In fact, the "handle" method is doing create/register on the plugin so maybe it will make more sense to say : "Plugin registered but not deployed" ? wdyt ?

Reply:
Maybe activated could be clearer. The plugin has been detected (ie: is present in the distributionà but has not been activated (because of the license) 🤷🏻‍♂️